tests/test_issues_generation.py

```diff
@@ -41,14 +40,15 @@ def test_when_validate_invalid_entity_with_valid_schema_should_return_errors(sel
             [],
             []
         )
-        mock_post.return_value.status = requests.codes.ok
+        mock_post.return_value.status = requests.codes['ok']
 
         current_folder = os.path.dirname(__file__)
-        with open(os.path.join(current_folder, "../resources/data_for_test_issues.json")) as test_data_file:
+        test_path = os.path.join(current_folder, "resources/data_for_test_issues.json")
```

Comment:
This variable is used only once, so that's why I used it inline in the method, like in line 49.
If you think it is cleaner, then we should also change it at line 49.